src/Symfony/Component/Messenger/Transport/Serialization/PhpSerializer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Serialization;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
+/**
+ * @author Ryan Weaver<ryan@symfonycasts.com>
+ *
+ * @experimental in 4.2
```

Comment:
Shouldn't this be `4.3`?

Reply:
Hmm, actually - not sure. The component is experimental in 4.2 - it won’t be (as of now in 4.3). I added this for consistency with the rest of the component. Either all will be updates at the same time to 4.3 or (hopefully) all will be removed.